pkgs/development/python-modules/fakeredis/default.nix

```diff
@@ -19,11 +20,13 @@ buildPythonPackage rec {
   version = "1.7.0";
   format = "setuptools";
 
-  disabled = pythonOlder "3.5";
+  disabled = pythonOlder "3.7";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "sha256-yb0S5DAzbL0+GJ+uDpHrmZl7k+dtv91u1n+jUtxoTHE=";
+  src = fetchFromGitHub {
+    owner = "jamesls";
+    repo = pname;
+    rev = version;
+    hash = "sha256-P6PUg9SY0Qshlvj+iV1xdrzVLJ9JXUV4cGHUynKO3m0=";
```

Comment:
Thanks for switching to the source build!

Reply:
Was need to use the patch.